eloquentjs/4_1_sum_of_a_range.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Returns the range of numbers from between start-end by incrementing by the value of step
```

Comment:
The logic of this function includes the start value which makes this line subtly incorrect because it implies an open interval. You should re-read the exercise since it specifies the behavior.